### DISCUSSION TITLE
Welcome to GitLearningRepo Discussions!

### DISCUSSION THREAD

**Discussion by Meghana250** [Meghana250/GitLearningRepo#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Meghana250** [comment]
@Bhavana-G Hello!

**Reply by Bhavana-G** [reply]
hi

**Reply by Meghana250** [reply]
https://code.tutsplus.com/tutorials/quick-tip-how-to-work-with-github-and-multiple-accounts--net-22574
